.github/actions/build/entrypoint.sh

```diff
@@ -12,6 +12,9 @@ echo "GITHUB_EVENT_PATH:" $GITHUB_EVENT_PATH
 echo "CONDA_PREFIX:" $CONDA_PREFIX
 echo "PWD:" `pwd`
 
+echo "Force clean of ccache"
```

Comment:
@klingaard You can delete the caches from here https://github.com/riscv-software-src/riscv-perf-model/actions/caches

Reply:
I know and I did.  Still have weird behavior.  Trying brute force to eliminate possibility.